README.md

```diff
@@ -210,6 +210,137 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
+request locally.
+
+Source: `app/mailers/prison_mailer.rb`
+
+- `GSI_SMTP_HOSTNAME`
+- `GSI_SMTP_PORT`
+- `GSI_SMTP_DOMAIN`
+
+#### To visitors
+
+These SMTP settings are used to send emails to visitors. Because these using SendGrid's SMTP server, they must be sent via SendGrid. You will need to configured them if you want to try and test email by submiting a booking request locally.
+
+Source: `config/environments/production.rb`
+
+
+- `SMTP_HOSTNAME`
+- `SMTP_PORT`
+- `SMTP_DOMAIN`
+- `SMTP_USERNAME`
+- `SMTP_PASSWORD`
+
+### Change default value of
+
+#### The Service Url
+
+This URL is used to configure the session store host and to generate links in emails. For development and test environment is set to `localhost`.
+
+Source: `config/environments/production.rb` and `config/initializers/session_store.rb`
+
+- `SERVICE_URL`
+
+#### The Prison Data File
+
+The default prison data file is `prison_data_staging.yml`, but you can configure a different one.
+
+Source: `config/application.rb`
+
+- `PRISON_DATA_FILE`
+
+#### The Message Encryptor Secret Key
+
+The key is a base 64-encoded value and is used to encrypt serialised data that are transmitted in URLs in the emails that are sent to prison staff (to process requests) and visitors (to view or cancel requests).
+
+Source: `config/initializers/encryptor.rb`
+
+- `MESSAGE_ENCRYPTOR_SECRET_KEY`
+
+#### The GovUk Start Page
+
+The page is using the default one `/prisoner`. If you want to change the default start page from `/prisoner` to something else, set this.
+
+Source: `config/routes.rb`
+
+- `GOVUK_START_PAGE`
+
+### Setting up
+
+#### The Google Analytics ID.
+
+Source: `config/application.rb`
+
+- `GA_TRACKING_ID`
+
+#### The Prison Estate IP
+
+Users on these IP addresses can access metrics without needing to log in, the prison booking admin pages and the prison staff info pages.
+
+Source: `config/application.rb`
+
+- `PRISON_ESTATE_IPS`
+
+#### The Zendesk
+
+These username and token need to be configured if you want to use feedback.
+
+Source: `config/initializers/zendesk.rb`
+
+- `ZENDESK_USERNAME`
+- `ZENDESK_TOKEN`
+
+#### The Session Secret Key
+
+This is used to cryptographically sign sessions both by Rails and by the Sidekiq queue maintenance interface.
+
+Source: `config/secrets.yml`
+
+- `SESSION_SECRET_KEY`
+
+#### The Trusted Users Access Key
+
+It's the key you pass in the URL when requesting the metrics page.
+
+Source: `config/application.rb`
+
+- `TRUSTED_USERS_ACCESS_KEY`
+
+#### The Elastic Search Url
+
+This url needs to be configured if you want to create new ELASTIC_CLIENT.
+
+Source: `config/initializers/metrics_logger.rb`
+
+- `ELASTICSEARCH_URL`
+
+#### The Stastsd
+
+These Host and Port needs to be configured if you want to create new STATSD_CLIENT.
+
+Source: `config/initializers/statsd_client.rb`
+
+- `STATSD_HOST`
+- `STATSD_PORT`
+
+### Smoke test configuration
+
+####Only the first two (local part and domain) are needed in order to
```

Comment:
@SteveMarshall I thought it would be better to be separate from the other list and have their own section.